inyoka_theme_default/static/style/inyoka/inyoka.less

```diff
@@ -231,6 +236,12 @@ input[type="text"], input[type="email"], input[type="password"], select {
   .form-control-focus();
 }
 
+table {
```

Comment:
just asking, as it could break something elsewhere: why applying the padding general?